src/types.ts

```diff
@@ -31,6 +31,7 @@ export interface V4TakerRequest extends BaseTakerRequest {
     protocolVersion: '4';
     txOrigin: string;
     isLastLook: boolean;
+    fee?: BigNumber;
```

Comment:
I think to keep this flexible we should add an additional param--`feeType: number`.

That way if we want to change what the fee represents, or even mix different types of fees for different situations, we can do so.